print("Printing")
raise Exception("TEST ERROR")